.gitignore

```diff
@@ -7,3 +7,4 @@
 .vagrant
 Vagrantfile
 /composer.lock
+.idea/
```

Comment:
No editor specific stuff in gitignore. Have a look at a global gitignore to ignore your favorite editors. https://help.github.com/articles/ignoring-files/